consul.yml

```diff
@@ -56,3 +56,30 @@ services:
       - CONSUL_BIND_INTERFACE=eth0
```

Comment:
Reduce number of consul instances to 3.
Try to have only 2 entries for consul - one with "master" and another one for agents joining consul1 with replicas parameter set to 2.